.travis.yml

```diff
@@ -43,11 +43,17 @@ before_script:
 
 script:
 - ./gradlew build connectedCheck test jacocoTestReport
+- bash assets/store_descriptions/generate_and_validate.sh
 
 after_success:
 - bash <(curl -s https://codecov.io/bash)
 
 before_deploy:
+- git config --local user.name "TravisCI"
```

Comment:
Copied from https://github.com/openhab/openhab-android/pull/525/files but I'm not sure if it works.